packages/block-library/src/pullquote/style.scss

```diff
@@ -55,7 +56,3 @@
 		}
 	}
 }
-
-.wp-block-pullquote cite {
-	color: inherit;
-}
```

Comment:
This style was [already defined](https://github.com/WordPress/gutenberg/blob/3c9a0207f51d731ff4d8cfbfa69daa16f797639f/packages/block-library/src/pullquote/style.scss#L8-L12) and removed due to duplication.